pkg/transfers/inbound/cleanup_test.go

```diff
@@ -43,3 +72,149 @@ func TestCleanupErr(t *testing.T) {
 		t.Errorf("unexpected deleted file: %s", agent.DeletedFile)
 	}
 }
+
+func Test_CleanupEmptyFiles_InboundPath_Success(t *testing.T) {
+	agent := &upload.MockAgent{}
+
+	dir, _ := ioutil.TempDir("", "clenaup-testing")
+	dl := &downloadedFiles{dir: dir}
+	defer dl.deleteFiles()
+
+	// write a test file to attempt deletion
+	path := filepath.Join(dl.dir, agent.InboundPath())
+	if err := os.MkdirAll(path, 0777); err != nil {
+		t.Fatal(err)
+	}
+
+	if err := ioutil.WriteFile(filepath.Join(path, "empty_file.ach"), []byte(""), 0644); err != nil {
+		t.Fatal(err)
+	}
+
+	files, err := ioutil.ReadDir(path)
+	if err != nil {
+		t.Fatal(err)
+	}
+	fileInfo := files[0]
+	mockTimeNow := fileInfo.ModTime().Add(10 * time.Minute)
+	minutesAfterToDelete := 5 * time.Minute
+
+	if err := CleanupEmptyFiles(log.NewNopLogger(), agent, dl, mockTimeNow, minutesAfterToDelete); err == nil {
+		t.Error("expected error")
+	}
+
+	if agent.DeletedFile != "inbound/empty_file.ach" {
```

Comment:
This one's more of a nit, but I think tests cases and their output are easier to read when testify is used. As an example, this block could be replaced by `assert.Equal(t, "inbound/empty_file.ach", agent.DeletedFile)`. If the test fails it will print out a nicely-formatted error message that lets you quickly see why the test failed.

Reply:
Good to know. I'll try it out and see how it goes. If it makes reading test output better, more informative, nicely formatted then that sounds great.